OmniConverter/Extensions/Audio/Renderers/XSynthRenderer.cs

```diff
@@ -159,7 +152,7 @@ public unsafe XSynthEngine(CSCore.WaveFormat waveFormat, Settings settings) : ba
 
             _groupOptions.stream_params = _streamParams;
             _groupOptions.channels = 16;
-            _groupOptions.fade_out_killing = CachedSettings.KilledNoteFading;
+            _groupOptions.fade_out_killing = !CachedSettings.KilledNoteFading;
             _groupOptions.use_threadpool = !(CachedSettings.MultiThreadedMode && CachedSettings.PerTrackMode);
 
             _layerCount = (ulong)Math.Round((decimal)(CachedSettings.MaxVoices / (128 * 16)), MidpointRounding.AwayFromZero);
```

Comment:
This is definitely not how the layers should be set. A single layer in XSynth is one voice per key per channel. So if the user has set a voice limit to 2048 or lower, the layer limit will be set to 1, which will sound considerably worse than an actual 2048 voice render (and that voice limit will not even be hit in most cases).

What I recommend is having different options for each synthesizer. So for example in the settings window when BASS is selected, the settings below the selection box will be related to BASS and if XSynth is selected they will be related to that respectively. XSynth also allows unlimited layers/voices for which there is no setting, and in order to hit high layer numbers, the voice count must be extremely high.

Reply:
This also applies to the interpolation and the threadpool settings. Since XSynth only supports nearest and linear (with pretty noticeable performance differences) I think that there should be a option for this (following the specific synthesizer options I mentioned above). The same goes for threadpooling which is also something the user should specify.

Reply:
Same goes for disabling audio effects (in the SF options) which is much different from BASS's because in XSynth it can disable the LPF which will alter the sound of a soundfont a lot if the user disables it thinking it only affects reverb and chorus like in BASS.